.gitmodules

```diff
@@ -1,5 +1,5 @@
-[submodule "pytorch_addons/third_party/googletest"]
-	path = pytorch_addons/third_party/googletest
+[submodule "pytorch_blade/third_party/googletest"]
```

Comment:
So where is the repo of pytorch_blade?

Reply:
`pytorch_blade` is now a folder of BladeDISC